File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_DomainsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_DomainsService : BaseService<Sys_Domains, ISys_DomainsRepository>
    , ISys_DomainsService
    {
    private readonly ISys_DomainsRepository _Sys_DomainsRepository;
    public Sys_DomainsService(ISys_DomainsRepository Sys_DomainsRepository) : base(Sys_DomainsRepository)
    {
    _Sys_DomainsRepository = Sys_DomainsRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_Domains.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_Domains } from './Sys_Domains.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_DomainsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Domain_Id', 'Domain_Code', 'Domain_Name', 'Parent_Id', 'Enabled', 'Description'];

    List: Sys_Domains[] | undefined;
    ListFilter: Sys_Domains[] | undefined;
    FormData!: Sys_Domains;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_Domains";
    }
}

